origin-contracts/package-lock.json

```diff
@@ -15372,6 +15373,14 @@
 			"resolved": "https://registry.npmjs.org/typedarray/-/typedarray-0.0.6.tgz",
 			"integrity": "sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c="
 		},
+		"typedarray-to-buffer": {
```

Comment:
@crazybuster I was repeatedly getting `Cannot find module 'typedarray-to-buffer'` until reverting this back to master. 🤷‍♂️